elementary/monitor/dbt_project_utils.py

```diff
@@ -9,42 +9,36 @@
 _DBT_PROJECT_FILENAME = "dbt_project.yml"
 _PACKAGES_FILENAME = "packages.yml"
 
-PATH = os.path.join(_MONITOR_DIR, "dbt_project")
+CLI_DBT_PROJECT_PATH = os.path.join(_MONITOR_DIR, "dbt_project")
 
-# Compatibility for previous dbt versions
-_MODULES_PATH = os.path.join(PATH, "dbt_modules")
-_PACKAGES_PATH = os.path.join(
-    PATH, os.environ.get("DBT_PACKAGES_FOLDER", "dbt_packages")
-)
 
-
-def is_dbt_package_up_to_date() -> bool:
-    installed_version = _get_installed_dbt_package_version()
+def is_dbt_package_up_to_date(project_dir: str) -> bool:
+    installed_version = _get_installed_dbt_package_version(project_dir)
     if installed_version is None:
         return False
 
-    required_version = _get_required_dbt_package_version()
+    required_version = _get_required_dbt_package_version(project_dir)
     # We're using a non dbt Hub requirement, such as Git or local.
     if not required_version:
         return True
 
     return installed_version == required_version
 
 
-def _get_elementary_package_path():
-    package_path = os.path.join(_PACKAGES_PATH, _DBT_PACKAGE_NAME)
+def _get_elementary_package_path(project_dir: str) -> Optional[str]:
+    package_path = os.path.join(project_dir, "dbt_project", _DBT_PACKAGE_NAME)
     if os.path.exists(package_path):
         return package_path
 
-    legacy_package_path = os.path.join(_MODULES_PATH, _DBT_PACKAGE_NAME)
+    legacy_package_path = os.path.join(project_dir, "dbt_modules", _DBT_PACKAGE_NAME)
```

Comment:
Let's deprecate it while we're at it.
(We already don't support _that_ old versions.)

Reply:
good point